torch/onnx/symbolic_helper.py

```diff
@@ -1262,6 +1262,49 @@ def _repeat_interleave_split_helper(g: jit_utils.GraphContext, self, reps, dim):
     return split_out if reps > 1 else [split_out]
 
 
+@_beartype.beartype
+def _repeat_interleave_single_value_helper(
+    g: jit_utils.GraphContext, self, repeats, dim
+):
+    from torch.onnx.symbolic_opset9 import flatten, unsqueeze
+
+    if not _is_tensor(repeats):
+        repeats = g.op("Constant", value_t=torch.LongTensor(repeats))
+
+    const_repeats: bool = _is_constant(repeats)
+    reps = _maybe_get_const(repeats, "t")
+
+    # Convert 'repeats' to 1-d if it is 0-d.
+    if _get_tensor_rank(repeats) == 0:
+        repeats = g.op("Reshape", repeats, g.op("Constant", value_t=torch.tensor([1])))
+
+    # Create a new dim of size 1, then resize it to be 'repeats' long, and finally collapse it.
```

Comment:
nit: expand or tile might be more precise.
```suggestion
    # Create a new dim of size 1, then expand it to be 'repeats' long, and finally collapse it.
```